### DISCUSSION TITLE
How to evaluate contributions of original features to latent components

### DISCUSSION THREAD

**Discussion by AlbertXTang** [AdaptiveMotorControlLab/CEBRA#148] [Q&A] [answered]
Thanks for your great work and contribution. I have a specific question regarding the evaluation of the contributions of original features to the reduced dimensions, analogous to how to assess the contribution of original features to the principal components after PCA. For instance, if I perform dimensionality reduction on the activity of 100 neurons and obtain 3 latent components, how can I evaluate the contribution of each of the 100 neurons to each latent components? Thank you!

**Comment by MMathisLab** [comment, marked as answer]
Thanks @AlbertXTang! This is something we are actively working on for a more elegant solution; the brute force is feature ablation (drop out neurons and train again, but a lot of compute). 

Here is a glimpse of the early work, but we will have more out soon: https://sslneurips23.github.io/paper_pdfs/paper_80.pdf

**Reply by AlbertXTang** [reply]
Thank you! We look forward to seeing the refined approach that your team comes up with!
